python/src/tests/tasks/test_list_genes.py

```diff
@@ -59,24 +59,24 @@ def test_throws_on_missing_parameters(self):
     def test_works_with_request(self):
         ListGenes(self.correct_desc)
 
-    def test_construct_request(self):
+    def test_format_request(self):
```

Comment:
renamed to _format_request to follow current convention. There were only 3 files with construct. all the others were _format. I'm fine either way but let's stick with one.